geomesa-features/geomesa-feature-kryo/src/main/scala/org/locationtech/geomesa/features/kryo/json/JsonAttrFunction.scala

```diff
@@ -0,0 +1,48 @@
+package org.locationtech.geomesa.features.kryo.json
+
+import com.typesafe.scalalogging.LazyLogging
+import org.geotools.filter.{AttributeExpressionImpl, FunctionExpressionImpl}
+import org.geotools.filter.capability.FunctionNameImpl
+import org.geotools.filter.capability.FunctionNameImpl.parameter
+import org.geotools.filter.expression.PropertyAccessors
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.expression.{PropertyName, VolatileFunction}
+
+class JsonAttrFunction extends FunctionExpressionImpl(
+  new FunctionNameImpl("jsonattr",
+    parameter("propertyName", classOf[PropertyName]),
```

Comment:
Oh snap... @elahrvivaz is right.  If this is a 'string', we can save on the changes to the FastPropertyName. 

*headdesk*